### PR TITLE
Migrate pg_locks to collector package

### PR DESCRIPTION
**Pull request by SuperQ**
Migrate the `pg_locks_count` query from `main` to the `collector` package.

### CODE REVIEW

**Comment by SuperQ** [comment]
CC @Sticksman

File: collector/pg_locks.go [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+		      ('accesssharelock'), 
+		      ('rowsharelock'), 
+		      ('rowexclusivelock'), 
+		      ('shareupdateexclusivelock'), 
+		      ('sharelock'), 
+		      ('sharerowexclusivelock'), 
+		      ('exclusivelock'), 
+		      ('accessexclusivelock'), 
+		      ('sireadlock')
+		  ) AS tmp(mode) CROSS
```
Comment (sysadmind) [inline collector/pg_locks.go:67]: Just a preference, but I think I like `CROSS` on the next line.

**Review by sysadmind — APPROVED** [review]
LGTM. One comment on formatting preference, but definitely not important.

File: collector/pg_locks.go (hunk 1 above)
Comment (SuperQ) [reply collector/pg_locks.go:67]: Yea, I don't have a preference on SQL formatters. Maybe we should pick and document a recommended one for the codebase.

**Review by Sticksman — APPROVED** [review]

**Pull request merged**